proto/rill/admin/v1/api.proto

```diff
@@ -2031,6 +2042,8 @@ message UnsubscribeReportRequest {
   string organization = 1;
   string project = 2;
   string name = 3;
+  optional string email = 4 [(validate.rules).string.email = true];
+  optional string slack_user = 5 [(validate.rules).string.email = true];
```

Comment:
The practice is to avoid `optional` unless unset and empty value have different meanings, which I don't think is the case here given that the empty string is not a valid email.

If `optional` is just for validation, I believe there's a validation rule you can set that will skip the other validation rules if the value is empty.

Reply:
yeah it was validation, let me see the rule you mentioned